### ISSUE TITLE
Recreate CI

### ISSUE THREAD

**Issue by dazuma** [googleapis/discovery-artifact-manager#438]
CI on this repo hasn't run since 2017, and the current scripts are no longer relevant. A bunch of things are currently being gutted from this repo (notably pretty much all client library generation and release scripts). I believe what will remain once that process is complete is:

* Updating the discovery doc cache
* Generating Apiary samples

Those two items will probably remain in this repo for some time, and it will be useful to cover their tests (if any) in CI. We should set that up.

**Comment by dazuma** [comment]
CI now running for the new discovery doc cache updater scripts.
